plugin-server/src/main/ingestion-queues/on-event-handler-consumer.ts

```diff
@@ -115,6 +98,7 @@ export const startAsyncWebhooksHandlerConsumer = async ({
 
     return {
         stop: async () => {
+            await actionManager.stop()
```

Comment:
don't we need this for onEvent too?

Reply:
Maybe? I couldn't see any "stop" hook to add it to though so I'm assuming it wasn't an issue before